src/main/java/springsamurais/toyswapbackend/model/User.java

```diff
@@ -1,3 +1,82 @@
 package springsamurais.toyswapbackend.model;
 
-public class User {}
+import jakarta.persistence.*;
+import org.antlr.v4.runtime.misc.NotNull;
+
+
+import java.util.List;
+
+@Entity
+//@Table(name = "user")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String name;
+
+    @Column(unique = true)
+    @NotNull
+    private String nickname;
+
+    private String location;
+
+
+    @Column
+    private List<Post> posts;
+
+    // Constructors
+    public User() {
+    }
+
+    public User(String name, String nickname, String location, List<Post> posts) {
+        this.name = name;
+        this.nickname = nickname;
+        this.location = location;
+        this.posts = posts;
+    }
```

Comment:
Initialize the Lis<Post> to avoid null pointer exception.

Reply:
Hi,  Update it , Thank you.